### ISSUE TITLE
[Feature Request] Add some i18n content

### ISSUE THREAD

**Issue by X7md** [termux/termux.github.io#59]
# The Idea:
Add some translate content, to other languages, such Arabic, Indonesian etc...

## Goal 
Official Resources to Termux, and can also reference to Termux Wiki.

**Comment by agnostic-apollo** [comment]
Yeah, when wiki is moved termux.org, the docs will support other locales, but users will have to translate themselves and send pull requests. There are auto translaters as well.

**Comment by AHSANNUL** [comment]
I'm new gaid me

**Comment by Vitahabtc2** [comment]
> # The Idea:
> Add some translate content, to other languages, such Arabic, Indonesian etc...
> 
> ## Goal 
> Official Resources to Termux, and can also reference to Termux Wiki. 

Get clone

**Comment by Vitahabtc2** [comment]
> # The Idea:
> Add some translate content, to other languages, such Arabic, Indonesian etc...
> 
> ## Goal 
> Official Resources to Termux, and can also reference to Termux Wiki. 

Get clone
